pages-e2e/features/_utils/getAssertVisible.ts

```diff
@@ -19,14 +19,21 @@ async function assertVisible(
 	page: Page,
 	...[selector, options]: Parameters<Page['locator']>
 ): Promise<void> {
-	const locator = page.locator(selector, options);
 	let isVisible = false;
-	try {
-		await locator.waitFor({
-			timeout: 500,
-		});
-	} catch {}
-	isVisible = await locator.isVisible();
+	for(const _attempt of [0, 1, 2, 3]) {
+		const locator = page.locator(selector, options);
+		try {
+			await locator.waitFor({
+				timeout: 200,
+			});
+		} catch {}
+		isVisible = await locator.isVisible();
+		if(isVisible) {
+			break;
+		} else {
+			await new Promise(resolve => setTimeout(resolve, 200));
```

Comment:
Do you need this extra timeout of 200ms, given that waitFor will wait for 200ms itself?

Reply:
Actually I am confused. The waitFor on the locator can return even if the selected thing is not visible?

Reply:
Yeah sorry I am not really sure, I might have left it there by mistake as I was trying things out 😓 

> The waitFor on the locator can return even if the selected thing is not visible?
yes, after the timeout it would throw if the element is not visible (/present)

Reply:
I've removed the `setTimeout`, I don't think it was right, thanks a lot for noticing it! ❤️